chain/chain/src/runtime/mod.rs

```diff
@@ -1233,6 +1233,26 @@ impl RuntimeAdapter for NightshadeRuntime {
     fn load_mem_tries_on_startup(&self, shard_uids: &[ShardUId]) -> Result<(), StorageError> {
         self.tries.load_mem_tries_for_enabled_shards(shard_uids)
     }
+
+    fn load_mem_trie_on_catchup(
```

Comment:
It's unclear to me why would we want to have these functions as part of the runtime adaptor and not access the tries directly?